### ISSUE TITLE
Create a site Traffic Light Classifier

### ISSUE THREAD

**Issue by knight-riders** [knight-riders/CarND-Capstone-Project#2]
Model selection, Training etc..
https://medium.com/@WuStangDan/step-by-step-tensorflow-object-detection-api-tutorial-part-1-selecting-a-model-a02b6aabe39e
https://becominghuman.ai/traffic-light-detection-tensorflow-api-c75fdbadac62
Using tensorboard : https://medium.com/@anthony_sarkis/tensorboard-quick-start-in-5-minutes-e3ec69f673af

Fun Fact: COCO dataset include Traffic Lights (minus their state)

Keep in mind, accuracy, inference time and the 100MB file size limit of GitHub.

Requirements:
TensorFlow v1.3, Keras v2.0.8 + Cuda 8 + Cudnn 6.0
